### ISSUE TITLE
Linking fails with GCC version 10

### ISSUE THREAD

**Issue by SoerenBusse** [rtbrick/bngblaster#80]
**Describe the bug**

When trying to compile bngblaster from source on a Debian bullseye, the linking fails with the following errors:
```
/usr/bin/ld: CMakeFiles/bngblaster.dir/src/bbl_a10nsp.c.o (symbol from plugin): in function `log_win':
(.text+0x0): multiple definition of `log_win'; CMakeFiles/bngblaster.dir/src/bbl.c.o (symbol from plugin):(.text+0x0): first defined here
/usr/bin/ld: CMakeFiles/bngblaster.dir/src/bbl_a10nsp.c.o (symbol from plugin): in function `log_win':
(.text+0x0): multiple definition of `stats_win'; CMakeFiles/bngblaster.dir/src/bbl.c.o (symbol from plugin):(.text+0x0): first defined here
```

**To Reproduce**
```
# Debian Bullseye 11.3
mkdir build && cd build
cmake /root/bngblaster-0.7.2 -DCMAKE_BUILD_TYPE=Release
cmake --build . --config Release
```

**Workaround**
When using gcc-9 everything works fine.
```
apt install gcc-9
cmake /root/bngblaster-0.7.2 -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc-9
cmake --build . --config Release
```

**Comment by GIC-de** [comment]
Could you please verify if the following patch would fix the compiler waring!

```diff
diff --git a/code/bngblaster/src/bbl.h b/code/bngblaster/src/bbl.h
index 92e1507..30372f8 100644
--- a/code/bngblaster/src/bbl.h
+++ b/code/bngblaster/src/bbl.h
@@ -39,8 +39,8 @@
 #include "bgp/bgp.h"
 #include "isis/isis.h"
 
-WINDOW *log_win;
-WINDOW *stats_win;
+extern WINDOW *log_win;
+extern WINDOW *stats_win;
 
 void
 enable_disable_traffic(bbl_ctx_s *ctx, bool status);

```

**Comment by SoerenBusse** [comment]
Yes this works :) Thank you for the fast reply.

**Comment by GIC-de** [comment]
Fixed in `dev` branch with target release `0.7.3`.
